.github/workflows/release.yml

```diff
@@ -16,21 +16,21 @@ jobs:
 
       - run: curl -sL https://git.io/goreleaser | bash -s -- --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.RELEASE_USER_TOKEN }}
+          GITHUB_TOKEN: "${{ secrets.RELEASE_USER_TOKEN }}"
 
   bump-version:
     name: Bump app version in Helm chart
     runs-on: ubuntu-latest
+    needs: ['build']
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
I don't think this is needed here as the whole release workflow (see line 3) only runs on tags.

Reply:
Good catch! Added :heavy_check_mark: